package.json

```diff
@@ -44,7 +44,7 @@
     "expect": "^1.13.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.8.2",
-    "flexboxgrid": "^6.3.0",
+    "flexboxgrid": "Zetoff/flexboxgrid#hidden-columns",
```

Comment:
Oh, I see! Maybe this PR should wait then until there are news about that feature.